polaris/ocean/vertical/zstar.py

```diff
@@ -113,7 +118,7 @@ def _compute_z_star_layer_thickness(restingThickness, ssh, bottomDepth,
     nVertLevels = restingThickness.sizes['nVertLevels']
     layerThickness = []
 
-    layerStretch = (ssh + bottomDepth) / bottomDepth
+    layerStretch = (ssh + bottomDepth) / numpy.sum(restingThickness, axis=1)
```

Comment:
Can we make this use xarray instead?
```suggestion
    layerStretch = (ssh + bottomDepth) / restingThickness.sum(dim='nVertLevels')
```